lib/modules/apostrophe-favicons-global/index.js

```diff
@@ -27,6 +27,13 @@ module.exports = {
         label: 'Favicon Link Tags',
         type: 'string',
         contextual: true,
+      },
+      {
+        name: 'aposFaviconRelationship',
```

Comment:
Adding an internal property to the schema but marking it contextual means that if someone *does* submit it via the API, it *will* set it.

You could mark it readOnly: true, but I would argue that internal properties that are never edited at all should simply not be in the schema. Lots of precedent for this: createdAt, updatedAt, searchSummary, various things that can't even be accurately described by our schemas.

Reply:
To be clear, all I'm asking you to do is remove it from the schema, it and it will still work if it's not in the schema.